packages/trpc/server/routers/viewer/me/updateProfile.handler.ts

```diff
@@ -152,7 +152,12 @@ export const updateProfileHandler = async ({ ctx, input }: UpdateProfileOptions)
   }
 
   // if defined AND a base 64 string, upload and update the avatar URL
-  if (input.avatarUrl && input.avatarUrl.startsWith("data:image/png;base64,")) {
+  if (
+    input.avatarUrl &&
+    (input.avatarUrl.startsWith("data:image/png;base64,") ||
+      input.avatarUrl.startsWith("data:image/jpeg;base64,") ||
+      input.avatarUrl.startsWith("data:image/jpg;base64,"))
```

Comment:
Let's create a reusable helper for this validation. I see it repeated in various places